auslib/test/conftest.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+
+from auslib.global_state import dbo
+
+
+@pytest.fixture(scope='session')
```

Comment:
It might be useful to add some comments here which describe how db_schema caches once for all the tests, avoiding doing a migration each time, while current_db_schema is used in each class setUp call to reuse it on a fresh DB.